src/scripts/run_certbot.sh

```diff
@@ -49,12 +49,39 @@ fi
 # $1: The name of the certificate (e.g. domain-rsa)
 # $2: String with all requested domains (e.g. -d domain.org -d www.domain.org)
 # $3: Type of key algorithm to use (rsa or ecdsa)
+# $4: The authenticator to use to solve the challenge
 get_certificate() {
-    info "Requesting an ${3^^} certificate for '${1}'"
+    local authenticator_params=
+    local challenge_type=
+
+    if [[ "$4" == dns-* ]] && \
```

Comment:
Just a small thing here, should we have "${4,,}" in all of these places for extra safety?

Reply:
Adding it, hesitated of doing it where we're calling the function, but it was done in the function for other parameters, so doing the same! :)